### PR TITLE
Update foreman-fog-proxmox to 0.6.0

### PR DESCRIPTION
**Pull request by tristanrobert**
For plugin updates, please indicate which repos this should be built into:

* [ ] Nightly
* [X] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: plugins/ruby-foreman-fog-proxmox/debian/gem.list [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_fog_proxmox-0.5.5.gem"
-GEMS="$GEMS https://rubygems.org/downloads/fog-proxmox-0.5.5.gem"
+GEMS="https://rubygems.org/downloads/foreman_fog_proxmox-*0.6.0.gem"
```
Comment (mmoll) [inline plugins/ruby-foreman-fog-proxmox/debian/gem.list]: that looks odd ;)

File: plugins/ruby-foreman-fog-proxmox/debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+ruby-foreman-fog-proxmox (0.6.0-1) stable; urgency=low
+
+  * 0.6.0 released. Update fog-proxmox dependency to 0.6.0
+
+ --  <>  Tue, 16 Apr 2019 15:12:09 +0200
```
Comment (mmoll) [inline plugins/ruby-foreman-fog-proxmox/debian/changelog]: This needs an update

File: plugins/ruby-foreman-fog-proxmox/debian/gem.list (hunk 1 above)
Comment (tristanrobert) [reply plugins/ruby-foreman-fog-proxmox/debian/gem.list]: oops! I see...

File: plugins/ruby-foreman-fog-proxmox/debian/changelog (hunk 2 above)
Comment (mmoll) [reply plugins/ruby-foreman-fog-proxmox/debian/changelog]: @tristanrobert Could you add your details here? I think the rest is good to go :rocket:.

**Comment by mmoll** [comment]
merged, merci @tristanrobert!

**Pull request merged**